src/client.ts

```diff
@@ -151,6 +116,38 @@ export class Client {
       });
   }
 
+  /**
+   *
+   * @param loadKeys
+   * @param persistKeys
+   * @param dag
+   */
+  public dagrun(loadKeys: string[] | null, persistKeys: string[] | null, dag: Dag): Promise<any> {
```

Comment:
This seems that we can do add a field `readonly `on `Dag`, Then we can merge `dagrun `and `dagrun_ro `into one method. WDYT?  :)

Reply:
I believe both have pros and cons. I'm fine with both :+1: . lets see what @lantiga or @hhsecond think about it :)

Reply:
@gkorland can you share your thoughts on both options?

Reply:
hi there @dengliming I believe we should keep the dagrun and dagrun_ro to have a direct association with the RedisAI commands. WDYT?

Reply:
agree. : )

Reply:
:+1: thanks for the fast reply =)